storage/src/main/java/com/tencent/rss/storage/api/ShuffleReader.java

```diff
@@ -26,5 +26,8 @@
 
   byte[] readData(long offset, int length);
 
+  byte[] readAllData();
```

Comment:
This api is unnecessary, readIndex(int size) is ok here,  return type can be changed if necessary.

Reply:
This is a readAll api or we can read all data in readData if the length is -1

Reply:
I also think we don't need this interface.

Reply:
readAll should be renamed as readIndex?

Reply:
Agree, I think we need a readIndexRawData api here, readAllData may break the sematic consistence of of this interface.

Reply:
I think `readIndex` may express the behavior more correctly that we want. Our `FileReader` present `ShuffleFIleReader`, not a common file reader. So the `FileReader` can have the `readIndex` interface.

Reply:
OK, add a readIndex api that return raw data of the index file.